Harvest.Api/Models/IdNameModel.cs

```diff
@@ -6,7 +6,7 @@ namespace Harvest.Api
 {
     public class IdNameModel
     {
-        public long Id { get; set; }
+        public long? Id { get; set; }
```

Comment:
It affect a lot of models, can you provide example when Id is null?

Reply:
could not replicate it - made it "long" again